MapboxSceneKit/TerrainNode.swift

```diff
@@ -90,6 +82,28 @@ open class TerrainNode: SCNNode {
         }
     }
 
+    private class func zoomLevelForBounds(minLat: CLLocationDegrees, maxLat: CLLocationDegrees, minLon: CLLocationDegrees, maxLon: CLLocationDegrees, tileSize: CGSize) -> Int {
```

Comment:
What do you think about additionally listening to `UIApplicationDidReceiveMemoryWarningNotification`, release the terrain images, decrease the zoom level and fetch everything once again?

Reply:
I didn't try this, but it should work.